Gems/DiffuseProbeGrid/Code/Source/Components/DiffuseProbeGridComponentController.cpp

```diff
@@ -482,6 +482,16 @@ namespace AZ
             m_featureProcessor->SetVisualizationSphereRadius(m_handle, m_configuration.m_visualizationSphereRadius);
         }
 
+        bool DiffuseProbeGridComponentController::CanBakeTextures()
+        {
+            if (!m_featureProcessor)
+            {
+                return false;
+            }
+
+            return m_featureProcessor->CanBakeTextures();
```

Comment:
Suggestion: could simplify a bit.
```suggestion
        return m_featureProcessor && m_featureProcessor->CanBakeTextures();
```